tests/DemosTest.php

```diff
@@ -189,6 +189,17 @@ class_exists(Utils::class) ? Utils::streamFor($body) : \GuzzleHttp\Psr7\stream_f
                 $body->rewind();
             }
 
+            $this->assertSame(
```

Comment:
I prefer assertions on the same line if not too long otherwise the LOC it too high and file overview is limited.

To your phpstan strugles:
```
ob_start();
try {
try {} catch {}
} finally {
ob...
}
```